keps/sig-apps/2232-suspend-jobs/README.md

```diff
@@ -280,7 +280,11 @@ Unit, integration, and end-to-end tests will be added to test that:
 ### Graduation Criteria
 
 #### Alpha -> Beta Graduation
-* Implemented feedback from alpha testers
+
+- [ ] Implemented feedback from alpha testers
+- [ ] Enable the `SuspendJob` feature gate by default
+- [ ] All unit, integration and end-to-end tests are run and they all pass
+- [ ] Documentation updated to reflect beta status
```

Comment:
graduation criteria is not a task list on how to graduate, but when to graduate.